jabgui/src/main/java/org/jabref/gui/preferences/JabRefGuiPreferences.java

```diff
@@ -674,17 +669,21 @@ private WorkspacePreferences getWorkspacePreferencesFromBackingStore(WorkspacePr
                 getStringList(SELECTED_SLR_CATALOGS));
     }
 
+    private UnlinkedFilesDialogPreferences getUnlinkedFilesDialogPreferencesFromBackingStore(UnlinkedFilesDialogPreferences defaults) {
+        return new UnlinkedFilesDialogPreferences(
+                get(UNLINKED_FILES_SELECTED_EXTENSION, defaults.getUnlinkedFilesSelectedExtension()),
+                DateRange.parse(get(UNLINKED_FILES_SELECTED_DATE_RANGE, defaults.getUnlinkedFilesSelectedDateRange().name())),
+                ExternalFileSorter.parse(get(UNLINKED_FILES_SELECTED_SORT, defaults.getUnlinkedFilesSelectedSort().name()))
+        );
+    }
+
```

Comment:
Please maintain order and structure in the file. Create another PR and move this below getUnlinkedFilesDialogPreferences